crypto/ec/ecp_secp384r1.c

```diff
@@ -0,0 +1,19594 @@
+         {UINT64_C(0xAF339C85296470A3), UINT64_C(0xF9603FCDAFD8203E),
+          UINT64_C(0x95D0535096763C28), UINT64_C(0x15445C16860EC831),
+          UINT64_C(0x2AFB87286867A323), UINT64_C(0x4B152D6D0C4838BF)}},
+        {{UINT64_C(0x45BA0E4F837CACBA), UINT64_C(0x7ADB38AEC0725275),
+          UINT64_C(0x19C82831942D3C28), UINT64_C(0x94F4731D6D0FE7DD),
+          UINT64_C(0xC3C07E134898F1E6), UINT64_C(0x76350EACED410B51)},
+         {UINT64_C(0x0FA8BECAF99AACFC), UINT64_C(0x2834D86F65FAF9CF),
+          UINT64_C(0x8E62846A6F3866AF), UINT64_C(0xDAA9BD4F3DFD6A2B),
+          UINT64_C(0xC27115BBA6132655), UINT64_C(0x83972DF7BD5A32C2)}},
+        {{UINT64_C(0xA330CB5BD513B825), UINT64_C(0xAE18B2D3EE37BEC3),
+          UINT64_C(0xFC3AB80AF780A902), UINT64_C(0xD7835BE2D607DDF1),
+          UINT64_C(0x8120F7675B6E4C2B), UINT64_C(0xAA8C385967E78CCB)},
+         {UINT64_C(0xA8DA8CE2AA0ED321), UINT64_C(0xCB8846FDD766341A),
+          UINT64_C(0xF2A342EE33DC9D9A), UINT64_C(0xA519E0BED0A18A80),
+          UINT64_C(0x9CDAA39CAF48DF4C), UINT64_C(0xA4B500CA7E0C19EE)}},
+        {{UINT64_C(0x83A7FD2F8217001B), UINT64_C(0x4F6FCF064296A8BA),
+          UINT64_C(0x7D74864391619927), UINT64_C(0x174C1075941E4D41),
+          UINT64_C(0x037EDEBDA64F5A6C), UINT64_C(0xCF64DB3A6E29DC56)},
+         {UINT64_C(0x150B3ACE37C0B9F4), UINT64_C(0x1323234A7168178B),
+          UINT64_C(0x1CE47014EF4D1879), UINT64_C(0xA22E374217FB4D5C),
+          UINT64_C(0x69B81822D985F794), UINT64_C(0x199C21C4081D7214)}},
+        {{UINT64_C(0x160BC7A18F04B4D2), UINT64_C(0x79CA81DDB10DE174),
+          UINT64_C(0xE2A280B02DA1E9C7), UINT64_C(0xB4F6BD991D6A0A29),
+          UINT64_C(0x57CF3EDD1C5B8F27), UINT64_C(0x7E34FC57158C2FD4)},
+         {UINT64_C(0x828CFD89CAC93459), UINT64_C(0x9E631B6FB7AF499F),
+          UINT64_C(0xF4DC8BC0DA26C135), UINT64_C(0x6128ED3937186735),
+          UINT64_C(0xBB45538B67BF0BA5), UINT64_C(0x1ADDD4C10064A3AB)}},
+    },
+    {
+        {{UINT64_C(0xC32730E8DD14D47E), UINT64_C(0xCDC1FD42C0F01E0F),
+          UINT64_C(0x2BACFDBF3F5CD846), UINT64_C(0x45F364167272D4DD),
+          UINT64_C(0xDD813A795EB75776), UINT64_C(0xB57885E450997BE2)},
+         {UINT64_C(0xDA054E2BDB8C9829), UINT64_C(0x4161D820AAB5A594),
+          UINT64_C(0x4C428F31026116A3), UINT64_C(0x372AF9A0DCD85E91),
+          UINT64_C(0xFDA6E903673ADC2D), UINT64_C(0x4526B8ACA8DB59E6)}},
+        {{UINT64_C(0x68FE359DE23A8472), UINT64_C(0x43EB12BD4CE3C101),
+          UINT64_C(0x0EC652C3FC704935), UINT64_C(0x1EEFF1F952E4E22D),
+          UINT64_C(0xBA6777CB083E3ADA), UINT64_C(0xAB52D7DC8BEFC871)},
+         {UINT64_C(0x4EDE689F497CBD59), UINT64_C(0xC8AE42B927577DD9),
+          UINT64_C(0xE0F080517AB83C27), UINT64_C(0x1F3D5F252C8C1F48),
+          UINT64_C(0x57991607AF241AAC), UINT64_C(0xC4458B0AB8A337E0)}},
+        {{UINT64_C(0x3DBB3FA651DD1BA9), UINT64_C(0xE53C1C4D545E960B),
+          UINT64_C(0x35AC6574793CE803), UINT64_C(0xB2697DC783DBCE4F),
+          UINT64_C(0
```

Comment:
All functions before this have some comment about what the function this, from here it doesn't.

Reply:
Thanks! Fixed in a1f6cbce3e595fa03752b782ede5ba95b5f5be8c